Library/Homebrew/brew.rb

```diff
@@ -178,6 +188,13 @@
   exit 1
 rescue MethodDeprecatedError => e
   onoe e
+
+  if Homebrew::EnvConfig.no_auto_update?
+    $stderr.puts "#{Tty.bold}Do not report this issue until you've run `brew update` and tried again.#{Tty.reset}"
```

Comment:
With this same error message thrice, it's probably appropriate to put it in a variable. 

```suggestion
    $stderr.puts "#{Tty.bold}You have disabled automatic updates. Do not report this issue until you've run `brew update` and tried again.#{Tty.reset}"
```